packages/ember-metal/lib/deprecate_property.js

```diff
@@ -23,7 +23,7 @@ import { set } from "ember-metal/property_set";
 
 export function deprecateProperty(object, deprecatedKey, newKey) {
   function deprecate() {
-    Ember.deprecate('Usage of `' + deprecatedKey + '` is deprecated, use `' + newKey + '` instead.');
+    Ember.deprecate(`Usage of "${deprecatedKey}" is deprecated, use "${newKey}" instead.`);
```

Comment:
This is interesting. I feel like they wrapped `deprecatedKey` in ` on purpose here (it's a keyword) - we should probably keep doing that.

```js
Ember.deprecate(`Usage of \`${deprecatedKey}\` is deprecated, use \`${newKey}\` instead.`);
```